src/Service/DocumentUploader.php

```diff
@@ -102,21 +102,22 @@ public function handleViewDocument(Document $document, bool $forceDownload = fal
             stream_copy_to_stream($fileStream, $outputStream);
         });
 
-        // We use document name as filename.
-        $filename = $document->getDocumentName();
+        // We use document name as filename - it may not contain / or \, so we replace those.
+        $documentName = $document->getDocumentName();
+        $ext = pathinfo($document->getFilename(), PATHINFO_EXTENSION);
```

Comment:
Why do you rename `$filename` til `$documentName`? “filename” is a much better name for a variable that contains the name of a file …

Reply:
A variable name should not reflect the how the value has been found, but should reflect what it's value is.